### PR TITLE
Fix storage exception

### PR DESCRIPTION
**Pull request by NeilBaner**
Fix for #177 .

### CODE REVIEW

File: docs/_config.yml [hunk 1]
```diff
@@ -1,3 +1,3 @@
-theme: jekyll-theme-cayman
+theme: jekyll-theme-hacker
```
Comment (yAOwzers) [inline docs/_config.yml]: HAHA DID YOU JUST CHANGE THE THEME BACK

**Comment by NeilBaner** [comment]
Not intentionally probably committed the copy on my machine which hadn't been updated lol. Oops.

**Review by franceneee — CHANGES_REQUESTED** [review]
please change the theme back to cayman, thank you HAHAHA

**Comment by NeilBaner** [comment]
> 
> 
> please change the theme back to cayman, thank you HAHAHA

Done.

**Review by franceneee — APPROVED** [review]

**Review by yAOwzers — APPROVED** [review]
NOICE

**Pull request merged**